trio/_tests/test_dtls.py

```diff
@@ -292,9 +289,8 @@ async def null_handler(_: object) -> None:  # pragma: no cover
 
 
 async def test_dtls_over_dgram_only() -> None:
-    with trio.socket.socket() as s:
-        with pytest.raises(ValueError):
-            DTLSEndpoint(s)
+    with trio.socket.socket() as s, pytest.raises(ValueError):
```

Comment:
this is definitely worse imho, even though the original isn't super pretty either

Reply:
Just looking at the diff for this particular one, yes I completely agree.

Reply:
Disabled combining with statements in 76d7413